types/regatta.model.ts

```diff
@@ -0,0 +1,26 @@
+export interface Regatta {
+	id: string;
+	venue_id: string;
+	name: string;
+	file: string;
+	start_date: Date;
+	end_date: Date;
+	race_type: string;
+	breaking_news: string;
+	correction_factor_settings: number; // Is dit een object?
+	correction_factor_type: null; // Wat is dit?
```

Comment:
Dit is een enum

Reply:
Waar kan ik die vinden?

Reply:
Ik zeg het verkeerd de type is uuid en staat in een table.
De settings is bool indien het hard waait of niet